CMakeLists.txt

```
@@ -122,15 +124,22 @@ if(GRIDKIT_ENABLE_IPOPT)
   include(FindIpopt)
 endif()
 if(GRIDKIT_ENABLE_SUNDIALS)
-  find_package(SUNDIALS 5.5.0 REQUIRED CONFIG 
+  find_package(SUNDIALS REQUIRED CONFIG 
                PATHS ${SUNDIALS_DIR}
                      ${SUNDIALS_DIR}/lib/cmake/sundials)
+  if("${SUNDIALS_VERSION}" VERSION_LESS "5.5.0")
+    message(FATAL_ERROR "GridKit requires SUNDIALS >= 5.5.0, got ${SUNDIALS_VERSION}")
+  endif()
```

```suggestion
  if("${SUNDIALS_VERSION}" VERSION_LESS "6.0.0")
    message(FATAL_ERROR "GridKit requires SUNDIALS >= 6.0.0, got ${SUNDIALS_VERSION}")
  endif()
```
Sundials have changed API as of version 6.x and there is no reason for GridKit to be backward compatible with older Sundials releases.